types/react-datepicker/index.d.ts

```diff
@@ -81,6 +86,7 @@ export interface ReactDatePickerProps<
     formatWeekDay?(day: string): React.ReactNode;
     formatWeekNumber?(date: Date): string | number;
     highlightDates?: Array<HighlightDates | Date> | undefined;
+    holidays?: Holiday[] | undefined;
```

Comment:
`| undefined` is probably not necessary because this prop has `?`.
However, I'm following the former contributes, since the other optional props generally have `| undefined`.

Reply:
FYI, `| undefined` is typically included because of the `exactOptionalPropertyTypes` config option: <https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes>